jaxrs/src/main/java/com/cerner/beadledom/jaxrs/JaxRsModule.java

```diff
@@ -19,5 +26,6 @@
   protected void configure() {
     bind(CorrelationIdFilter.class).toProvider(CorrelationIdFilterProvider.class)
         .in(Singleton.class);
+    bind(ForwardedHeaderFilter.class).in(Singleton.class);
```

Comment:
Do we have any concerns around installing this by default?

Reply:
Yeah, we should. It would be safer to just let the clients install it if they want to.

Reply:
I removed the default install, and added docs in the jaxrs.rst file: https://github.com/cerner/beadledom/pull/149/commits/5bce5a5cc4b901cf9b67fa7953618919bb44b789